tokio-postgres/src/lib.rs

```diff
@@ -142,6 +142,7 @@ pub use crate::to_statement::ToStatement;
 pub use crate::transaction::Transaction;
 pub use crate::transaction_builder::{IsolationLevel, TransactionBuilder};
 use crate::types::ToSql;
+pub use fallible_iterator::FallibleIterator;
```

Comment:
This makes it consistent with the `postgres` crate

```suggestion
pub use fallible_iterator;
```